spring-petclinic/pom.xml

```diff
@@ -748,21 +443,21 @@
 			<plugin>
```

Comment:
Based on what we talked and the purpose of the quickstart ( "is to show how to take a Spring app and deploy it on EAP/WildFly") we can also put some of our flavours on it. This means that we can remove the maven-compiler-plugin and use the maven.compiler.source and maven.compiler.target properties to make it aligned with Java SE 8. This is how we do it for the other quickstarts.